tests/TestCodeGen.hs

```diff
@@ -113,10 +130,12 @@ simpleDecodeDotProto =
 
 -- E.g. dumpAST ["test-files"] "test_proto.proto"
 dumpAST :: [FilePath] -> FilePath -> IO ()
-dumpAST incs fp = do
-  Right (dp, tc) <- readDotProtoWithContext incs fp
-  let Right src = renderHsModuleForDotProto mempty dp tc
-  putStrLn src
+dumpAST incs fp = act >>= either (putStrLn . show) putStrLn
```

Comment:
Maybe use something like `throws` here, so that if the result is a `Left` the message is printed to `stderr` instead of `stdout`.  i.e.:

```haskell
dumpAST incs fp = throws act >>= putStrLn
```

Reply:
I couldn't find any package that defines `throws`, so I just changed it to `either throwIO putStrLn`